### ISSUE TITLE
UnsatisfiedLinkError during sdk init

### ISSUE THREAD

**Issue by FrancoisBlavoet** [mapbox/mapbox-gl-native#11390]
**Platform:** Android, Galaxy J7(2016)
**Mapbox SDK version:** Mapbox/5.4.1

### Steps to trigger behavior
Some of our users are encountering the following crash : 
Fatal Exception: java.lang.UnsatisfiedLinkError
No implementation found for void com.mapbox.mapboxsdk.net.NativeConnectivityListener.initialize() (tried Java_com_mapbox_mapboxsdk_net_NativeConnectivityListener_initialize and Java_com_mapbox_mapboxsdk_net_NativeConnectivityListener_initialize__)

On our end, it is triggered by `Mapbox.getInstance()`. Unfortunately we can't really do anything to prevent this internal error

**Comment by tobrun** [comment]
Thank you for the report, sadly this issue is not actionable for us as something during installation or loading of the native library went wrong. That being said, with the upcoming 6.0.0 release we are allowing to change the component responsible for loading the native library. This will allow to change the platform default way of loading so files with something as [Relinker](https://github.com/KeepSafe/ReLinker) or [SOLoader](https://github.com/facebook/SoLoader) and could potentially make loading of so files more robust.

**Comment by tobrun** [comment]
The API for above is:

```
LibraryLoader#setLibraryLoader(LibraryLoader libraryLoader)
```

**Comment by cpienovi** [comment]
@FrancoisBlavoet did you find a way to prevent this from happening?
we're experiencing the exact same thing

**Comment by leocarona** [comment]
I'm using Mapbox 6.4.0 and also getting logs of users experiencing same exception.

**Comment by Ajojo44** [comment]
Same on version 7.0.0 of Mapbox :/

**Comment by tobrun** [comment]
With https://github.com/mapbox/mapbox-gl-native/pull/13974 and thus 7.3.0, we ship relinker as default library loader.

**Comment by subodhsri** [comment]
Same issue I am getting with 7.0.7

**Comment by tobrun** [comment]
this isn't an actual issue, see https://github.com/mapbox/mapbox-gl-native/issues/15331#issuecomment-539857586 for more context.

**Comment by bc67da8d** [comment]
This issue is duplicate and I solved it in #14428
